docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,171 @@
+   Indicates a successful call.
+
+=== Example
+
+Retrieve the first five cases with the `phishing` tag, in ascending order by
+last update time:
+
+[source,sh]
+--------------------------------------------------
+GET api/cases/_find?page=1&perPage=5&sortField=updatedAt&sortOrder=asc&tags=phishing
+--------------------------------------------------
+// KIBANA
+
+The API returns a JSON object listing the retrieved cases. For example:
+
+[source,json]
+--------------------------------------------------
+{
+  "page": 1,
+  "per_page": 5,
+  "total": 2,
+  "cases": [
+    {
+      "id": "abed3a70-71bd-11ea-a0b2-c51ea50a58e2",
+      "version": "WzExMCwxXQ==",
+      "comments": [],
+      "totalComment": 0,
+      "totalAlerts": 0,
+      "title": "The Long Game",
+      "tags": [
+        "windows",
+        "phishing"
+      ],
+      "description": "Windows 95",
+      "settings": {
+        "syncAlerts": true
+      },
+      "owner": "securitySolution",
+      "closed_at": null,
+      "closed_by": null,
+      "created_at": "2022-03-29T13:03:23.533Z",
+      "created_by": {
+        "email": "rhustler@email.com",
+        "full_name": "Rat Hustler",
+        "username": "rhustler"
+      },
+      "status": "open",
+      "updated_at": null,
+      "updated_by": null,
+      "connector": {
+        "id": "131d4448-abe0-4789-939d-8ef60680b498",
+        "name": "My connector",
+        "type": ".jira",
+        "fields": {
+          "issueType": "10006",
+          "priority": null,
+        }
+      }
+      "external_service": null,
+    },
+    {
+      "id": "a18b38a0-71b0-11ea-a0b2-c51ea50a58e2",
+      "version": "Wzk4LDFd",
+      "comments": [],
+      "totalComment": 0,
+      "totalAlerts": 0,
+      "title": "This case will self-destruct in 5 seconds",
+      "tags": [
+        "phishing",
+        "social engineering",
+        "bubblegum"
+      ],
+      "description": "James Bond clicked on a highly suspicious email banner advertising cheap holidays for underpaid civil servants. Operation bubblegum is active. Repeat - operation bubblegum is now active!",
+      "settings": {
+        "syncAlerts": false
+      },
+      "owner": "cases",
+      "closed_at": null,
+      "closed_by": null,
+      "created_at": "2022-03-29T11:30:02.658Z",
+      "created_by": {
+        "email": "ahunley@imf.usa.gov",
+        "full_name": "Alan Hunley",
+        "username": "ahunley"
+      },
+      "status": "open",
+      "updated_at": "2022-03-29T12:01:50.244Z",
+      "updated_by": {
+        "full_name": "Classified",
+        "email": "classified@hms.oo.gov.uk",
+        "username": "M"
+      },
+      "connector": {
+        "id": "131d4448-abe0-4789-939d-8ef60680b498",
+        "name": "My connector",
+        "type": ".resilient",
+        "fields": {
+          "issueTypes": [13],
+          "severityCode": 6,
+        }
+      },
+      "external_service": null,
+    }
+  ],
+  "count_open_cases": 2,
+  "count_in_progress_cases":0,
```

Comment:
>   "count_in_progress_cases":0,

This property was also missing from the original docs in https://www.elastic.co/guide/en/security/master/cases-api-find-cases.html so if that was intentional, let me know!